text/0000-named-tracers.md

```diff
@@ -0,0 +1,89 @@
+# Named Tracers
+
+**Status:** `proposed`
+
+_Creating Tracers using a factory mechanism and naming those Tracers in accordance with the library that provides the instrumentation for a traced component._
+
+## Suggested reading
+
+* [Proposal: Tracer Components](https://github.com/open-telemetry/opentelemetry-specification/issues/10)
+* [Global Instance discussions](https://github.com/open-telemetry/opentelemetry-specification/labels/global%20instance)
+* [Proposal: Add a version resource](https://github.com/open-telemetry/oteps/pull/38)
+
+## Motivation
+
+The mechanism of "Named Tracers" proposed here is motivated by following scenarios:
+
+* As a consumer of OpenTelemetry instrumentation libraries, there is currently no possibility of influencing the amount of the data produced by such libraries. Instrumentation libraries can easily "spam" backend systems, deliver bogus data or - in the worst case - crash or slow down applications. These problems might even occur suddenly in production environments caused by external factors such as increasing load or unexpected input data.
+
+* If a library hasn't implemented semantic conventions correctly or those conventions change over time, it's currently hard to interpret and sanitize these data selectively. The produced Spans cannot associated with those instrumentation libraries later.
```

Comment:
this part is confusing - how is semver related to sanitizing data?

Reply:
This is refers to the "semantic conventions" part of the specification. I'll a link to it (https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md) here.